docs/internals/deprecation.txt

```diff
@@ -30,6 +30,9 @@ details on these changes.
 
 * ``Field.remote_field.to`` attribute will be removed.
 
+* The ``on_delete`` argument for ``ForeignKey``, ``ForeignObject``, and

```

Comment:
That's fine with me. Sometimes we have put private APIs through deprecations if we think or have evidence that they are commonly used.